### ISSUE TITLE
Can no longer reenter corpse as a ghost on headless

### ISSUE THREAD

**Issue by fomalsd** [unitystation/unitystation#2443]
# Description
When attempting to reenter corpse on our dev server, camera freezes and you lose control of ghost. Ghost disappears, too.

**Comment by chairbender** [comment]
and get a bunch of NRE spam from playersync.update, playersync.syncronize

**Comment by chairbender** [comment]
I think this is caused by this:
https://github.com/vis2k/Mirror/issues/962

which was fixed 4 days ago

**Comment by chairbender** [comment]
reproducible locally btw (with client)

**Comment by chairbender** [comment]
Alright, so here's the deal:

When I upgraded mirror to the release that had this fix, I encountered a new issue - SyncVar hooks were being called before Awake which is definitely not how it's supposed to work. Even when I upgraded to the latest release on github I had this issue.

Mirror devs told me it's not a good idea to use the github release because it's not stable, so it's better to stick to the asset store release.

They said this fix would go into the asset store release in a matter of days, so I think we should just plan on using that.
